go-controller/pkg/util/util.go

```diff
@@ -376,3 +376,9 @@ func GenerateId(length int) string {
 	}
 	return string(b)
 }
+
+// IsWildcard checks if the domain name is wildcard.
+func IsWildcard(dnsName string) bool {
+	trimmedString := strings.TrimPrefix(dnsName, "*")
```

Comment:
Maybe just strings.Contains("*") will be easier to read?

Reply:
Would the following be more apt?
```go
func IsWildcard(dnsName string) bool {
	return strings.Index(dnsName, "*") == 0
}
```

Reply:
maybe if you want to make sure it's a prefix, `strings.HasPrefix` is the easiest?